src/runtime/PythonEngine.cs

```diff
@@ -127,6 +127,10 @@ public static string PythonPath
             }
         }
 
+        public static Version MinSupportedVersion => new(3, 7);
+        public static Version MaxSupportedVersion => new(3, 10, int.MaxValue, int.MaxValue);
```

Comment:
Maybe something more "readable", like 999 instead of maxint?

Reply:
I thought of that, or maybe do `MaxSupportedVersionExclusive`. Alternatively, we can only leave the `IsSupportedVersion` check and provide an error string in a separate property.